tests/github/test_runner.py

```diff
@@ -22,14 +22,14 @@ def test_runner_webhooks_check(self):
         runner = Runner()
         runner.github.github_conf_dir_path = valid_dir_path
 
-        checks = ["CKV_GITHUB_6","CKV_GITHUB_7"]
+        checks = ["CKV_GITHUB_6", "CKV_GITHUB_7"]
         report = runner.run(
             root_folder=valid_dir_path,
             runner_filter=RunnerFilter(checks=checks)
         )
         self.assertEqual(len(report.failed_checks), 1)
         self.assertEqual(report.parsing_errors, [])
-        self.assertEqual(len(report.passed_checks), 3)
+        self.assertEqual(len(report.passed_checks), 2)
```

Comment:
Did we lose a check here? I would expect that if we have less passing checks we'll have more failed checks or skipped checks.

Reply:
We didn't lose a check. initially, there was an error - see bug 2 in my description.